docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -40,6 +40,22 @@ At a minimum, ``hive.metastore.uri`` must be configured:
     connector.name=iceberg
     hive.metastore.uri=thrift://localhost:9083
 
+Iceberg configuration properties
+--------------------------------
+
+================================================== ============================================================ ============
```

Comment:
Convert this to a list table please

Reply:
I assume this just means adding them as different sub-sections. Let me know if it's otherwise.

Reply:
Oh .. sorry.. here is an example of a list table - 

Cache Configuration Parameters
in
https://github.com/trinodb/trino/blob/master/docs/src/main/sphinx/connector/hive-caching.rst

Reply:
Done. Can you review this once again?